pkg/catalog/egress.go

```diff
@@ -105,6 +105,10 @@ func (mc *MeshCatalog) GetEgressTrafficPolicy(serviceIdentity identity.ServiceId
 	}, nil
 }
 
+func getErrCode(e errcode.ErrCode) string {
```

Comment:
should this be moved to a util ? I assume it can be used across the code

Reply:
Accidentally merged 2 branches. fixed.